src/main/java/com/shigotosagashi/jobdomainservice/controller/JobController.java

```diff
@@ -0,0 +1,49 @@
+package com.shigotosagashi.jobdomainservice.controller;
+
+import com.shigotosagashi.jobdomainservice.dto.out.ExampleEntityDto;
+import com.shigotosagashi.jobdomainservice.repository.ExampleEntityRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.persistence.EntityNotFoundException;
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/job")
+public class JobController {
+
+  @Autowired
+  private ExampleEntityRepository exampleEntityRepository;
+
+  @PostMapping("/save")
```

Comment:
Como estamos construindo uma API RESTful, acredito que possamos utilizar a semântica de cada verbo pra determinar o comportamento de cada endpoint. Dessa forma, não precisaríamos deixar explícito os seguintes endpoints: `save`, `update`, `find` e `delete`.

Reply:
I get your point. My question is, what if we end up adding more and more endpoints to this service? How would we differentiate one method from another?
Any suggestions?

Reply:
If I understand right, we can use explicit endpoint when necessary. Just don't use for these simple functions because is redundant.

I don't see problems on both ways.